cli/jhipster.js

```diff
@@ -35,24 +35,33 @@ if (!semver.satisfies(currentNodeVersion, minimumNodeVersion)) {
     process.exit(1);
 }
 
-requireCLI();
+let preferGlobal = false;
+if (process.argv.length >= 3) {
```

Comment:
Yes I saw commander use in cli.js and thought about extracting some common code but it could make things even more complex because whatever file I put this common code in I might have to choose between local and global version. So it might be simpler to have jhipster.js depends as little as possible on other files.